webdash/style.scss

```diff
@@ -213,3 +208,8 @@ main h6 {
     border-color: #FF3D00;
   }
 }
+
+.btn-cancel-simple {
+  @extend .btn-cancel;
+  margin-left: 20px;
+}
```

Comment:
I kept that button small on purpose. You didn't like it?

Reply:
Actually the size was better.  My gripe with it was that it didn't match the style of the rest of the website.